distribution/src/resources/core-site-example.xml

```diff
@@ -30,4 +30,14 @@
         <value>ENTER_YOUR_SECRETKEY</value>
     </property>
 
+    <property>
```

Comment:
Please comment out this section and add comment explaining that user should use one of those.